integration/integration.go

```diff
@@ -234,6 +241,14 @@ func (i *Integration) Logger() log.Logger {
 	return i.logger
 }
 
+// CreateUniqueID will generate an md5 string from integration arguments
+// to unique identify the integration instance.
+func (i *Integration) CreateUniqueID() string {
+	h := md5.New()
+	h.Write([]byte(fmt.Sprintf("%v", i.args)))
```

Comment:
are the configurations passed as environment variables inside args? I'm thinking of the conifguration sample for k8s that the integrations have it only uses env-vars to pass conifgurations and no cli args.